tests/e2e/workloads/jenkins/test_jenkins_node_reboot.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import (
+    E2ETest, workloads, ignore_leftovers
```

Comment:
Any reason for using ignore_leftovers here?
If you are using this because of  rook-ceph-drain-canary pod then you can ignore that specific pods using the ignore_leftover_label